samples/graph-meeting-notification/nodejs/index.js

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Import required pckages
+const path = require('path');
+// Read botFilePath and botFileSecret from .env file.
+const ENV_FILE = path.join(__dirname, '.env');
+require('dotenv').config({ path: ENV_FILE });
+const restify = require('restify');
+const express = require('express');
+const { MeetingNotficationBot } = require('./bots/meeting-notification-bot');
+//const { MeetingNotficationBot } = require('./bots/meeting-notification-bot');
+const { DecryptionHelper } = require("./helper/decryption-helper");
```

Comment:
These line of code looks messy, kindly provide proper line spacing after file header and remove unwanted commented code.